ekip/everykid/static/css/style.css

```diff
@@ -54,40 +59,78 @@ nav.top-links {
       padding-right: 0;
       border: none; }
 
+#educators .educators-link {
+  color: #F7D27F; }
+
 nav.main-nav {
-  width: 175px;
-  position: absolute; }
+  width: 190px;
+  position: absolute;
+  top: 40px; }
+  nav.main-nav img {
+    width: 150px;
+    margin: 0 0 10px 0; }
   nav.main-nav ul {
     display: inline-block; }
   nav.main-nav li {
     background: #fff;
     color: #456D97;
-    padding: 7px 12px;
     margin: 0 0 10px 0;
+    border: 2px solid #1E7197;
     text-transform: uppercase;
     font-size: 13px;
     font-family: "Superclarendon", serif;
     text-align: center;
     display: block; }
     nav.main-nav li a {
+      width: inherit;
+      height: inherit;
+      padding: 7px 12px;
       text-decoration: none;
-      color: #456D97; }
+      color: #456D97;
+      display: block; }
+      nav.main-nav li a:hover {
+        background: #1E7197;
+        color: #fff; }
+
+#landing .main-nav {
+  top: 95px; }
+  #landing .main-nav img {
+    display: none; }
+
+.mountain {
+  background: url(../img/backgrounds/mountain.jpg) no-repeat center center fixed;
```

Comment:
We'll have to figure out what to do with these URLs later. When we move to production they will be hosted on S3, and those URLs can't be relative at that point.